lib/pom.xml

```diff
@@ -141,6 +142,77 @@
+                <version>1.4</version>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>create</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <getRevisionOnlyOnce>true</getRevisionOnlyOnce>
+                    <shortRevisionLength>7</shortRevisionLength>
+                    <timestampFormat>{0,date,yyyyMMdd}</timestampFormat>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>build-helper-maven-plugin</artifactId>
+                <version>1.10</version>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>parse-version</goal>
+                            <goal>regex-property</goal>
+                        </goals>
+                        <configuration>
+                            <name>rpm.release_extra</name>
+                            <value>${parsedVersion.qualifier}git${buildNumber}</value>
+                            <regex>SNAPSHOT</regex>
+                            <replacement>\${timestamp}</replacement>
+                            <failIfNoMatch>false</failIfNoMatch>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>rpm-maven-plugin</artifactId>
+                <version>2.1.4</version>
+                <executions>
+                    <execution>
+                        <id>generate-rpm</id>
+                        <goals>
+                            <goal>rpm</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <group>esbtools</group>
+                    <release>${rpm.release_number}.${rpm.release_extra}</release>
+                    <mappings>
+                        <mapping>
+                            <directory>/usr/share/java/lightblue/${project.artifactId}</directory>
+                            <sources>
+                                <source>
+                                    <location>${project.build.directory}/${project.build.finalName}.${project.packaging}</location>
+                                </source>
+                            </sources>
+                            <dependency>
+                                <excludes>
+                                    <exclude>com.redhat.lightblue:lightblue-core-config</exclude>
+                                    <exclude>com.redhat.lightblue:lightblue-core-crud</exclude>
+                                    <exclude>com.redhat.lightblue:lightblue-core-metadata</exclude>
```

Comment:
This still includes libraries that I don't think you want, such as jackson-annotations and jsr305.

You may consider specifying a single include rather than a bunch of excludes.
```xml
<includes>
  <include>${project.groupId}:*</include>
</includes>
```